packages/mui-material/src/Alert/Alert.js

```diff
@@ -43,6 +43,7 @@ const AlertRoot = styled(Paper, {
 })(({ theme, ownerState }) => {
   const getColor = theme.palette.mode === 'light' ? darken : lighten;
   const getBackgroundColor = theme.palette.mode === 'light' ? lighten : darken;
+  const getContrastText = theme.palette.mode === 'light' ? lighten : darken;
```

Comment:
This returns the same as `getBackgroundColor`, so is redundant. IT also doesn't do what its name suggests.

We have `getContrastText` in the theme. @mnajdova is it also in System? I couldn't find it documented for either theme or system, but I believe it's what @reramjiawan was referring to.

Reply:
We only have it in the `theme.palette` as part of the `@mui/material`'s theme.